uiowa/uiowa
# Add click-a11y.js from UIDS4

Resolves https://github.com/uiowa/uiowa/issues/6138

# How to test

- Review code
- Test functionality 
- Confirm unlinked cards/banners are not linked
- Confirm cards are still linked after using views exposed filters

```
ddev blt ds --site=housing.uiowa.edu
```
Go to /residence-halls and use the filters and verify that the cards are still clickable.  Compare with prod version. 
```
ddev blt ds --site=sandbox.uiowa.edu
```
- Test click behavior on banners /https://sandbox.uiowa.ddev.site/banners-imagesvideos-w-black-and-white-gradients
- Test click behavior on cards https://sandbox.uiowa.ddev.site/timeline, https://sandbox.uiowa.ddev.site/cards
